### ISSUE TITLE
lava room invincible chain?

### ISSUE THREAD

**Issue by nr1995** [nipkownix/re4_tweaks#407]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.1: https://www.re4hd.com/?p=9552)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.1 SFV: [BIO4-HDProject1.1-SFV.zip](https://github.com/emoose/re4-research/files/9207370/BIO4-HDProject1.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

Hi, noob here.
Is there a reason I can't shoot the chain links of the first dragon on the right side of the lava room?
I didn't try all weapons, but with the rifle,
I can shoot the one on the left before it starts moving, but not the right.
Anyone else getting this behaviour?
Thanks!

![Imgur](https://i.imgur.com/3tzjGfZ.jpg)

**Comment by linkthehylian** [comment]
Seems to be just fine on my end?

https://user-images.githubusercontent.com/20933012/209904227-d9610c71-a5e7-4ae1-a688-4940eb0db82b.mp4

**Comment by nr1995** [comment]
@linkthehylian
As soon as you walk in the room,
take out the wooden rifle, and try to break those chains,
on the same dragon as in your video there.
For me the shots don't interact with the chain at all.

**Comment by LethalPlacebo** [comment]
I believe this is how it's always been, the statues need to be "active" before the chains can be shot. This requires you getting close to the set of stairs.

**Comment by nr1995** [comment]
@LethalPlacebo   But you can break the chains on the left dragon before it starts moving.

**Comment by LethalPlacebo** [comment]
> @LethalPlacebo But you can break the chains on the left dragon before it starts moving.

Yeah, I don't know why that is. I'll try check later today if this is original behavior in the GC version.

**Comment by LethalPlacebo** [comment]
Okay I actually just tried this on the Steam version instead and I am able to destroy both chains just fine, without even having to get near the steps. I can destroy them as soon as enter the room. Have you installed any other mods or perhaps messed with settings in the trainer?

**Comment by nr1995** [comment]
@LethalPlacebo  The tweaks, but not the trainer section, no.
Have you tried it with a wooden rifle?

**Comment by LethalPlacebo** [comment]
> @LethalPlacebo The tweaks, but not the trainer section, no. No mods besides RE4HD v1.1 Have you tried it with a wooden rifle?

Hm, yes, the same as me then, just the HD Project and latest version of RE4 Tweaks.

Yeah, I tested it both with the default handgun at default stats and the bolt-action Rifle. Both broke the chain just fine.

**Comment by nr1995** [comment]
@LethalPlacebo  I'll try and make a video for you tomorrow.
